### PR TITLE
Merge bugfix into stable

### PR DESCRIPTION
**Pull request by jmichelp**

### CODE REVIEW

**Review by kaczmarczyck — APPROVED** [review]

**Comment by jmichelp** [comment]
It seems I forgot to rebase before.
And I can't use Github UI to rebase before getting all approvals...

**Review by ia0 — APPROVED** [review]

**Review by itsh5 — APPROVED** [review]

**Review by itsh5 — APPROVED** [review]

**Pull request merged**
